src/bundle/Resources/translations/custom_tags.en.yaml

```diff
@@ -0,0 +1,26 @@
+# ezyoutube
+ezrichtext.custom_tags.ezyoutube.label: YouTube
```

Comment:
IMHO this file should be converted to xliff format which is used by all our repositories.